common/src/main/java/org/apache/gravitino/dto/util/DTOConverters.java

```diff
@@ -1188,4 +1244,25 @@ public static JobTemplate fromDTO(JobTemplateDTO jobTemplateDTO) {
             "Unsupported job template type: " + jobTemplateDTO.jobType());
     }
   }
+
+  /**
+   * Converts a PolicyContentDTO to a PolicyContent.
+   *
+   * @param policyContentDTO The policy content DTO to be converted.
+   * @return The policy content.
+   */
+  public static PolicyContent fromDTO(PolicyContentDTO policyContentDTO) {
+    if (policyContentDTO == null) {
+      return null;
+    }
+
+    if (policyContentDTO instanceof PolicyContentDTO.CustomContentDTO) {
+      PolicyContentDTO.CustomContentDTO customContentDTO =
+          (PolicyContentDTO.CustomContentDTO) policyContentDTO;
+      return PolicyContents.custom(customContentDTO.customRules(), customContentDTO.properties());
+    }
+
+    throw new IllegalArgumentException(
```

Comment:
Why does `toDTO` supports two cases `PolicyContentDTO` and `olicyContents.CustomContent` and one here?